web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -35,13 +39,33 @@ interface RuleGroup {
   interval: number;
 }
 
+const kvSearchRule = new KVSearch({
+  shouldSort: true,
+  indexedKeys: ['name', 'labels', ['labels', /.*/]],
```

Comment:
I wonder if we want to be able to search in annotations as well. But it would probably make the index significantly more expensive (not sure how much of a problem that is), since annotations can be quite long. So maybe we can omit it for now.

Reply:
I was thinking about searching in the expression too.
But yeah indexing is quite expensive. Probably the search required to be more optimized around this usage.